### PR TITLE
Fix a typo in README.md

### PR DESCRIPTION
**Pull request by CleanMachine1**
snifferred isn't a word

Sniffed is the correct usage of the word 

also sh seems to be just shortened to shell, and shell is normally used from what I see

### CODE REVIEW

**Comment by bl-ue** [comment]
I think `sh` is better than `shell`, but here's some additional context: https://github.com/nodejs/remark-preset-lint-node/pull/136

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ As mentioned above, if `https://` is not specified, connection will be redirecte
 
 If you want to build your own image, instead of directly pull the pre-built one, clone this repository, and run `docker build` command, with build-arg `MAXMIND_LICENSE_KEY`, you need to provide your own MaxMind license key from https://www.maxmind.com/en/account, it's **free**.
 
-```sh
+```shell
```
Comment (PeterDaveHello) [inline README.md:118]: It's quote common to just use `sh` I think, and also, there are multiple places use `sh` in this file, so prefer to keep it just `sh`, would you mind to help revert this change? Thanks.

**Review by PeterDaveHello — CHANGES_REQUESTED** [review]
Nice catch 👍 Just a small part need to be revised.

File: README.md (hunk 1 above)
Comment (CleanMachine1) [reply README.md:118]: Of course

Comment (CleanMachine1) [reply README.md]: https://github.com/PeterDaveHello/ipinfo.tw/pull/9/commits/1b453fd6c9b474505a933d9baf117cdab82d700f

I would have changed them all to shell if the change was to go ahead.

**Review by PeterDaveHello — APPROVED** [review]
